ihatemoney/web.py

```diff
@@ -654,16 +653,15 @@ def list_bills():
     # set the last selected payer as default choice if exists
     if "last_selected_payer" in session:
         bill_form.payer.data = session["last_selected_payer"]
-    # Preload the "owers" relationship for all bills
-    bills = (
-        g.project.get_bills()
-        .options(orm.subqueryload(Bill.owers))
-        .paginate(per_page=100, error_out=True)
+
+    # Each item will be (weight_sum, Bill) tuple
+    weights_bills = g.project.get_bill_weights_ordered().paginate(
```

Comment:
should we change the naming for this to be `bills_with_weights`, or `weighted_bills`? `weight_bills` isn't clear enough for me.

Reply:
I went with `weighted_bills`.